src/App.js

```diff
@@ -74,6 +74,7 @@ function App() {
     const isEquals = input.match(/=/);
     const isDecimal = input.match(/./);
     const isClear = input.match(/AC/);
+    const isFloat = input.match(/^\d*\.?\d*$/);
```

Comment:
This regex seems to allow matching against single integer values, e.g. `3`. Is that intended? The name `isFloat` makes me think this regex would only allow matching on floating point numbers, but not simple integers.